### DISCUSSION TITLE
Can't install as PWA

### DISCUSSION THREAD

**Discussion by mausch** [kasmtech/KasmVNC#206] [Q&A] [answered]
Hi, I have `kasmweb/core-ubuntu-jammy` running in Kubernetes. It works great except I can't get the option to install it as PWA...
As mentioned https://kasmweb.com/docs/latest/guide/pwa.html it's served from HTTPS with a valid certificate.
Has anyone got this to work? Or maybe PWA isn't included in the core images?

Many thanks.

**Comment by j-travis** [comment, marked as answer]
The PWA is only available when you are using the Kasm Workspaces platform, not the standalone images, or KasmVNC standalone
